### ISSUE TITLE
AIPD factor missing in action

### ISSUE THREAD

**Issue by martinholmer** [open-source-economics/taxdata#65]
I've merged the current master branch --- which includes PR #63 --- into the PR #64 branch.
Then I did this with the current PR #64 code:
```
cd taxdata/stage1
python stage1.py
python factors_finalprep.py
```
The `Stage_I_factors.csv` output is as always, but when I compare the `factors_finalprep.py` output file (now called growfactors.csv) with the `growfactors.csv` in taxcalc PR #1178 (which I made from the old `StageIFactors.csv` file in taxcalc) there is only one set of differences: the AIPD factor is not in the `growfactors.csv` file generated by the `factors_finalprep.py` script.

@andersonfrailey, Where in taxdata is the AIPD factor added in?

**Comment by martinholmer** [comment]
I've now merged PR #64 into the taxdata master branch, so the problem reported in issue #65 can be reproduced with the taxdata master branch.

**Comment by martinholmer** [comment]
I've now merged Tax-Calculator PR #1178 into the Tax-Calculator master branch, so the correct version of `growfactors.csv` is now on the Tax-Calculator master branch.

The problem is that taxdata stage1 logic does not produce a `growfactors.csv` file that includes the AIPD factor.

@andersonfrailey, Can you prepare a pull request that fixes this problem?

**Comment by andersonfrailey** [comment]
@martinholmer I'll get to work on that now.

**Comment by martinholmer** [comment]
taxdata issue #65 has been resolved by merged pull request #67.
